test/activities/GradeCandidateEntity.js

```diff
@@ -41,4 +43,70 @@ describe('GradeCandidateCollectionEntity', () => {
 			expect(associateGradeSpy.returnValues[0]).to.be.a('promise');
```

Comment:
I know you didn't write this, but I don't think this test is actually doing anything, and doesn't really need to use a spy.
All it does is verifies that the `associateGrade` returns a promise, which by itself is not particularly useful, and it could have been verified by just checking the return value of `associateGrade`. Ideally tests like this would be using fetchMock to verify that the right API is called with the correct parameters.

Reply:
Also, this will actually try and make an API calls which can screw up other tests that are trying to verify API calls using mocks.

Reply:
fixed